### ISSUE TITLE
insight module missing

### ISSUE THREAD

**Issue by jcrpto** [Rudy-Tmc/backupAndRestoreInsightCloud#3]
Hi Rudy, did you forget to commit the `insight` module?
Backup script fails on the initial line, where it tries to import such a module.

```
$ python backupInsight.py
Traceback (most recent call last):
  File "/Users/user/src/insights/backupAndRestoreInsightCloud/backupInsight.py", line 1, in <module>
    from insight import insightConnect
ModuleNotFoundError: No module named 'insight'
```

**Comment by Rudy-Tmc** [comment]
Hi Jonas, currently I'm on holiday. So I don't have access to my laptop.

**Comment by Rudy-Tmc** [comment]
But isn't it here? https://github.com/Rudy-Tmc/backupAndRestoreInsightCloud/blob/main/insight.py

**Comment by jcrpto** [comment]
You're right!

Something strange happens when I clone the repo. The files get deleted. Never ran into this sort of problem with git before, so didn't even check the github repo view to validate what I cloned was what I saw.

Seems to be something odd with the insight.py filename. Could definitely be a local issue.

```
$ git clone https://github.com/Rudy-Tmc/backupAndRestoreInsightCloud
Cloning into 'backupAndRestoreInsightCloud'...
remote: Enumerating objects: 66, done.
remote: Counting objects: 100% (66/66), done.
remote: Compressing objects: 100% (47/47), done.
remote: Total 66 (delta 41), reused 42 (delta 19), pack-reused 0
Receiving objects: 100% (66/66), 34.15 KiB | 1.90 MiB/s, done.
Resolving deltas: 100% (41/41), done.
error: unable to create symlink insight.py: File name too long
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

$ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	deleted:    README.md
	deleted:    backupInsight-config.json
	deleted:    backupInsight.py
	deleted:    importObjectSchema-config.json
	deleted:    importObjectSchema.py
	deleted:    insight.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	README.md
	backupInsight-config.json
	backupInsight.py
	importObjectSchema-config.json
	importObjectSchema.py
```

You can close this. I'll investigate whatever may be wrong with my macos git. At least I can re-create the expected workspace locally manually and not rely on git clone.

**Comment by jcrpto** [comment]
In case anyone else runs into the same problem. I was able to repro this 100% on different linux machines as well.
The problem isn't with clone but with checking out. It is related to symlinks (don't know the root cause).

To fix, either disable symlinks in git before cloning, or disable them only for this project.
I chose the latter.

1. clone the repo, and get a failed half-checked out clone on the filesystem.
2. enter the project folder `cd backupAndRestoreInsightCloud`
3. Disable symlinks for the project `git config  core.symlinks false`
4. Do an initial checkout again: `git checkout --force`
